compiler/rustc_middle/src/mir/pretty.rs

```diff
@@ -462,19 +462,29 @@ impl<'tcx> Visitor<'tcx> for ExtraComments<'tcx> {
             let val = match literal {
                 ConstantKind::Ty(ct) => match ct.kind() {
```

Comment:
i think we now just want to pretty print the `ty` constant like a type system constant.
`

Reply:
generally, we should now go ahead and accept the churn from printing mir and ty constants in different ways.

(and update the comments in the function accordingly)

Reply:
I'll do that in another PR.